Tasks/Xcode/xcode.ts

```diff
@@ -338,7 +338,7 @@ async function run() {
                         exportMethod = await sign.getProvisioningProfileType(embeddedProvProfiles[0]);
                         tl.debug('Using export method = ' + exportMethod);
                     }
-                    if (!exportMethod) {
+                    if (!exportMethod && sdk === 'iphoneos') {
```

Comment:
This check isn't fullproof. Our Xcode template includes sdk, but it is optional.